server/forms.py

```diff
@@ -0,0 +1,58 @@
+from django import forms
+from .models import User, Player, Membership, Vaccination
+
+GENDER_CHOICES = [('male', 'Male'), ('female', 'Female'), ('other', 'Other')]
+
+
+class UserForm(forms.ModelForm):
+    class Meta:
+        model = User
+        fields = ['email', 'phone']
+
+
+class PlayerForm(forms.ModelForm):
+    date_of_birth = forms.DateField(widget=forms.DateInput(attrs={'type' : 'date'}))
+    gender = forms.ChoiceField(choices=GENDER_CHOICES)
+    other_gender = forms.CharField(required=False, max_length=20)
```

Comment:
Should we add an additional `other_gender` field on the model?

Reply:
we could add, now it is just "Other" in the model enum

Reply:
What is the form input `other_gender` doing? What's the point of having an input in the form, if nothing is getting saved to the DB? It has to be a field on the model.

Reply:
remove the input field from the form